cinema/tests/test_movie_api.py

```diff
@@ -123,7 +122,7 @@
 
         self.assertEqual(res.status_code, status.HTTP_201_CREATED)
         movie = Movie.objects.get(title="Title")
-        self.assertFalse(movie.image)
+        self.assertTrue(movie.image)
```

Comment:
The test case correctly checks if the image is associated with the created movie. This ensures that the image upload functionality works as expected.